packages/controller/src/resources/app/app.ts

```diff
@@ -20,24 +20,23 @@ import {
   Deployment,
   ClusterRoleBinding,
   ServiceAccount,
-  Namespace,
   Service,
   Secret,
   withPodAntiAffinityRequired
 } from "@opstrace/kubernetes";
 import { KubeConfig } from "@kubernetes/client-node";
 import { State } from "../../reducer";
-import {
-  generateSecretValue,
-  getControllerConfig
-} from "../../helpers";
+import { generateSecretValue, getControllerConfig } from "../../helpers";
 import { DockerImages } from "@opstrace/controller-config";
+import { getCertSecretCopy } from "../utils";
 
 export function OpstraceApplicationResources(
   state: State,
   kubeConfig: KubeConfig,
   namespace: string,
-  domain: string
+  domain: string,
+  ingressNameSpace: string,
```

Comment:
NameSpace is not camel-cased. For consistency in the codebase please use:

```suggestion
  ingressNamespace: string,
```